packages/dev/biome-config/configs/biome-package.jsonc

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "../../../node_modules/@biomejs/biome/configuration_schema.json",
+  "extends": ["./biome-base.jsonc"],
```

Comment:
unfortunately, extending from the same package is not supported by biome, it needs to be combined on a consumer side

Reply:
I was experiencing it right now 😓 then I guess that I should remove this remove on on package do 
```
extends: [base, package]
```
right?

Reply:
yup!